frontend/packages/console-app/src/components/detect-namespace/namespace.ts

```diff
@@ -19,43 +19,26 @@ type NamespaceContextType = {
 
 export const NamespaceContext = React.createContext<NamespaceContextType>({});
 
+const useUrlNamespace = () => getNamespace(useLocation().pathname);
+
 export const useValuesForNamespaceContext = () => {
-  const { pathname } = useLocation();
-  const urlNamespace = getNamespace(pathname);
-  const [lastNamespace, setLastNamespace, lastNamespaceLoaded] = useLastNamespace();
+  const urlNamespace = useUrlNamespace();
+  const [currentNamespace, setCurrentNamespace] = React.useState<string>(urlNamespace);
```

Comment:
@jerolimov sorry getting confused here is between what is `activeNamespce` vs `currentNamespace`, is it just a rename?

Reply:
That's right. Changed back to `activeNamespace`. This makes more sense with the `useActiveNamespace` hook to consume this data.